tests/unit/components/sl-calendar-test.js

```diff
@@ -11,32 +13,6 @@ moduleForComponent( 'sl-calendar', 'Unit | Component | sl calendar', {
     unit: true
 });
 
-test( 'Default class name is present', function( assert ) {
```

Comment:
Is the use of the `.sl-calendar` selector necessary in every test where it's currently used?  A goal in tests should be to use the least amount of (setup) code as possible/necessary?  I'm putting this comment here since you mentioned it rather than on each occurrence of use.